app/components/admins/all_collections_component.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module Dashboard
+module Admins
```

Comment:
Can this be `Admin` (singular) instead?

Reply:
Updated here and in other filepaths in order to align with https://github.com/sul-dlss/happy-heron/pull/2421